### PR TITLE
updated version check for cli

### PR DESCRIPTION
**Pull request by trapacska**
### New Pull Request Checklist

*Please mark the points which you did / accept.*

- [ ] __I will not move an already shared step version's tag to another commit__
- [ ] I read the [Step Development Guideline](https://github.com/bitrise-io/bitrise/blob/master/_docs/step-development-guideline.md)
- [ ] I have a test for my Step, which can be run with `bitrise run test` (in the step's repository)
- [ ] I did run `bitrise run audit-this-step` (in the step's repository - note, if you don't have this workflow in your `bitrise.yml`, [you can copy it from the step template](https://github.com/bitrise-steplib/step-template/blob/master/bitrise.yml).)
- [ ] I read and accept the [Abandoned Step policy](https://github.com/bitrise-io/bitrise-steplib#abandoned-step-policy)

### CODE REVIEW

File: handlers/validate/validators/collection/collection.go [hunk 1]
```diff
@@ -83,23 +82,25 @@ func getTestableCLIVersionDownloadURLs() ([]string, error) {
...
-		if version, err := version.NewVersion(release.TagName); err == nil && version.GreaterThan(latestSupportedVersion) && !version.Equal(unsupportedVersion) {
+		if version, err := version.NewVersion(release.TagName); err == nil && version.GreaterThan(latestSupportedVersion) && !isUnsupportedVersion(version, "1.39.0", "1.39.1") {
 			releaseTags = append(releaseTags, fmt.Sprintf("https://github.com/bitrise-io/bitrise/releases/download/%s/bitrise-%s-%s", release.TagName, os, arch))
 		}
 	}
 
 	return releaseTags, nil
 }
 
+func isUnsupportedVersion(ve *version.Version, unsupportedVersions ...string) bool {
```
Comment (godrei) [inline handlers/validate/validators/collection/collection.go]: pls write a quick unit test for this function

File: handlers/validate/validators/collection/collection_test.go [hunk 2]
```diff
@@ -0,0 +1,53 @@
...
+		unsupportedVersions []string
+		want                bool
+	}{
+		{
+			name:                "no unsupported versions",
+			unsupportedVersions: nil,
+			want:                false,
+		},
+		{
+			name:                "one supported version",
```
Comment (godrei) [inline handlers/validate/validators/collection/collection_test.go]: **un**supported

Comment (trapacska) [reply handlers/validate/validators/collection/collection_test.go]: nope

File: handlers/validate/validators/collection/collection_test.go [hunk 3]
```diff
@@ -0,0 +1,53 @@
...
+func Test_isUnsupportedVersion(t *testing.T) {
+	v, err := version.NewVersion("1.2.3")
+	require.NoError(t, err)
+
+	tests := []struct {
+		name                string
+		unsupportedVersions []string
+		want                bool
+	}{
+		{
```
Comment (godrei) [inline handlers/validate/validators/collection/collection_test.go]: I suggest to differentiate 3 test cases:
- nil unsupported versions (edge case check)
- 2 unsupported version + version included 
- 2 unsupported version + version NOT included
